consensus/istanbul/backend/api.go

```diff
@@ -63,41 +67,65 @@ func (api *API) GetSnapshotAtHash(hash common.Hash) (*Snapshot, error) {
 	if header == nil {
 		return nil, errUnknownBlock
 	}
+
 	return api.istanbul.snapshot(api.chain, header.Number.Uint64(), header.Hash(), nil, false)
 }
 
 // GetValidators retrieves the list of authorized validators with the given block number.
 func (api *API) GetValidators(number *rpc.BlockNumber) ([]common.Address, error) {
-	snap, err := api.GetSnapshot(number)
+	previousBlockNumber := rpc.BlockNumber(number.Int64() - 1)
```

Comment:
Is this okay? It seems that `PendingBlockNumber.Int64()` returns `-2`.

Reply:
Never mind. Just checked https://github.com/klaytn/klaytn/pull/1468/files#r913307162

Reply:
Thank you for your review. 
I fixed the problem that you mentioned. 
Reflected in https://github.com/klaytn/klaytn/pull/1468/commits/307820c288c4d7d1871a57d4b03f849deb46fdd0.
Thanks!